public/js/user.js

```diff
@@ -359,6 +360,62 @@ apos.define('apostrophe-workflow', {
       });
     };
 
+    self.enableForceExportRelated = function() {
```

Comment:
Logic is similar to the existing feature inside the "force export" modal for also force exporting related documents, however there is no modal for exporting the original doc in this case (we're not doing that), so the logic for iterating over the related ones goes right here.